docs/devel/owners.md

```diff
@@ -9,6 +9,8 @@ will serve as the approvers for code to be submitted to these parts of the repos
 are not necessarily expected to do the first code review for all commits to these areas, but they are
 required to approve changes before they can be merged.
 
+**Note** The Kubernetes project has a hiatus on adding new approvers to OWNERS files. At this time we are [adding more reviewers](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr%20%22Curating%20owners%3A%22%20) to take the load off of the current set of approvers and once we have had a chance to flush this out for a release we will begin adding new approvers once again. This is planned for the post Kubernetes 1.6.0 has been released
```

Comment:
I hate nitpicking tiny PRs, but the last sentence here does not parse as English.